census/ui_app/EntryForm.jsx

```diff
@@ -141,6 +144,7 @@ const EntryForm = React.createClass({
                   qsSchema={this.yourKnowledgeQSSchema}
                   questions={this.yourKnowledgeQuestions}
                   answers={this.props.answers.aboutYouAnswers}
+                  readonly={readonly}
```

Comment:
Yeah, that's a boolean passed to the components. The components use its value to determine whether to populate the `readOnlyOpts` object for use by textareas and inputs. So custom components get the boolean, and the final rendered textareas and inputs get the `readOnly='readonly'` attribute key/value.